components/AvalancheForecastZoneMap.tsx

```diff
@@ -76,7 +76,6 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
   const avalancheCenterMapRegionBounds: RegionBounds = zones
     ? zones.reduce((accumulator, currentValue) => updateBoundsToContain(accumulator, toLatLngList(currentValue.geometry)), defaultAvalancheCenterMapRegionBounds)
```

Comment:
using `defaultAvalancheCenterMapRegionBounds` as the accumulator will modify that constant in place, is that the right thing to do?

maybe `defaultAvalancheCenterMapRegionBounds` should instead be a function that returns an empty bounds object every time it's called.

Reply:
or do I have this wrong, and `updateBoundsToContain` doesn't modify its argument in place? if that's the case then ignore my feedback!

Reply:
Yeah it is written specifically not to mutate the input

Reply:
(and there's a test for that) :)